docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -6,142 +6,31 @@ toc_min_heading_level: 2
 toc_max_heading_level: 2
 ---
 
-import TabItem from '@theme/TabItem';
```

Comment:
**Note for reviewer** - When I filed https://greatexpectations.atlassian.net/browse/DOC-922, I imagined I'd add a new tab on this page for Databricks. But I ended up deciding to remove the provider-specific tabs from this page for a few reasons

- Maintenance burden - we missed adding one new datasource already. 
- Page scope - this page went into too much detail about adding a new Data Source given that its focus is supposed to be managing Data Assets. The "Connect GX Cloud to ..." pages already cover adding a new Data Source, so it was duplicative to have the content on this page as well. d

Reply:
I think this is a good choice!